app/views/saml_idp/shared/saml_post_binding.html.slim

```diff
@@ -13,6 +13,7 @@ html
         p = t('.no_js')
 
         form action=action_url method='POST' id='saml-post-binding'
+          = hidden_field_tag('csp_uris', csp_uris) if Rails.env.test?
```

Comment:
Is there a reason we couldn't test `expect(page.response_headers['Content-Security-Policy'])` instead of checking for this?